### PR TITLE
Towards module-acceptance procedure

### PR DESCRIPTION
**Pull request by MikeTaylor**

### CODE REVIEW

File: MODULE_EVALUATION.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* [x] Installation documentation is included
+  * -_note: read more at https://github.com/folio-org/mod-search/blob/master/README.md_
+* [x] Personal data form is completed, accurate, and provided as `PERSONAL_DATA_DISCLOSURE.md` file
+* [x] Sensitive and environment-specific information is not checked into git repository
+* [x] Module is written in a language and framework from the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page
+* [x] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn't been accepted yet_
+* [x] Module gracefully handles the absence of third party systems or related configuration
+* [ ] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication
+* [x] Uses [officially supported](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) build tools
+* [ ] Unit tests have 80% coverage or greater, and are based on [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)
```
Comment (adamdickmeiss) [inline MODULE_EVALUATION.md:27]: It's 87 now. Will be higher soon.

File: MODULE_EVALUATION.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* [ ] Module provides reference data (if applicable), e.g. if there is a controlled vocabulary where the module requires at least one value
+* [ ] If provided, integration (API) tests must be written in an [officially approved technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)
+  * -_note: while it's strongly recommended that modules implement integration tests, it's not a requirement_
+  * -_note: these tests are defined in https://github.com/folio-org/folio-integration-tests_
+* [x] Data is segregated by tenant at the storage layer
+* [x] The module doesn't access data in DB schemas other than its own and public
+* [x] The module responds with a tenant's content based on x-okapi-tenant header
+* [x] Standard GET `/admin/health` endpoint returning a 200 response
+  * -_note: read more at https://wiki.folio.org/display/DD/Back+End+Module+Health+Check+Protocol_
+* [ ] High Availability (HA) compliant
```
Comment (adamdickmeiss) [inline MODULE_EVALUATION.md:47]: You can tick this one too.

File: MODULE_EVALUATION.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* [x] Module descriptor MUST include interface requirements for all consumed APIs
+* [x] Third party dependencies use an Apache 2.0 compatible license
+* [x] Installation documentation is included
+  * -_note: read more at https://github.com/folio-org/mod-search/blob/master/README.md_
+* [x] Personal data form is completed, accurate, and provided as `PERSONAL_DATA_DISCLOSURE.md` file
+* [x] Sensitive and environment-specific information is not checked into git repository
+* [x] Module is written in a language and framework from the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page
+* [x] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn't been accepted yet_
+* [x] Module gracefully handles the absence of third party systems or related configuration
+* [ ] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication
```
Comment (adamdickmeiss) [inline MODULE_EVALUATION.md:25]: Passed.

File: MODULE_EVALUATION.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## Backend
+* [x] Module's repository includes a compliant Module Descriptor
+  * -_note: read more at https://github.com/folio-org/okapi/blob/master/okapi-core/src/main/raml/ModuleDescriptor.json_
+* [x] Environment vars are documented in the ModuleDescriptor
+  * -_note: read more at [https://wiki.folio.org/pages/viewpage.action?pageId=65110683](https://wiki.folio.org/pages/viewpage.action?pageId=65110683)_
+* [x] If a module provides interfaces intended to be consumed by other FOLIO Modules, they must be defined in the Module Descriptor "provides" section
+* [x] All API endpoints are documented in RAML or OpenAPI
+* [x] All API endpoints protected with appropriate permissions as per the following guidelines and recommendations, e.g. avoid using `*.all` permissions, all necessary module permissions are assigned, etc.
+  * -_note: read more at https://dev.folio.org/guidelines/naming-conventions/ and https://wiki.folio.org/display/DD/Permission+Set+Guidelines_
+* [ ] Module provides reference data (if applicable), e.g. if there is a controlled vocabulary where the module requires at least one value
```
Comment (adamdickmeiss) [inline MODULE_EVALUATION.md:38]: There are no reference data for this module, so fine.

**Review by adamdickmeiss — COMMENTED** [review]
Some can be ticked.

**Comment by MikeTaylor** [comment]
Thanks, Adam. I updated the checklist accordingly.

**Comment by MikeTaylor** [comment]
That only leaves integration tests. If you're planning to do those, fine; if not, we can mark it as INAPPLICABLE.

**Review by adamdickmeiss — APPROVED** [review]

**Pull request merged**